docs/flow/modin/core/dataframe/base/partitioning/axis_partition.rst

```diff
@@ -4,6 +4,10 @@ BaseDataframeAxisPartition
 The class is base for any axis partition class and serves as the last level on which
 operations that were conveyed from the partition manager are being performed on an entire column or row.
 
+**Note**: ``modin.core.dataframe.base`` intentionally does not describe any particular partition interface,
```

Comment:
Not sure I follow your logic here. Also, see the comment above on adding a base partition manager and a base partition classes.

Reply:
The logic here was we don't have any base partition interface because we don't deal with them directly at Modin Frame level - it's up to concrete implementation/storage format to decide on the partition interface

Reply:
makes sense